goldens-prerelease test fails with ` TypeError: MessageToJson() got an unexpected keyword argument 'including_default_value_fields'`

The pre-release test failed in PR https://github.com/googleapis/gapic-generator-python/pull/1932 around the time of a protobuf pre-release `protobuf==5.26.0rc2`. https://pypi.org/project/protobuf/5.26.0rc2/

The test is failing because the `including_default_value_fields` argument was renamed in `MessageToJson`.

The default value of `including_default_value_fields` is `False`, which is the same value that is being set, so we can remove it from the call to resolve the issue.

version 3.19.6 - default value of `including_default_value_fields` is `False`
https://github.com/protocolbuffers/protobuf/blob/5cba162a5d93f8df786d828621019e03e50edb4f/python/google/protobuf/json_format.py#L92

At the time that the argument was removed - default value of `including_default_value_fields` is `False`
https://github.com/protocolbuffers/protobuf/commit/26995798757fbfef5cf6648610848e389db1fecf#diff-8de817c14d6a087981503c9aea38730b1b3e98f4e306db5ff9d525c7c304f234

```
=================================== FAILURES ===================================
_________ test_generate_access_token_rest[GenerateAccessTokenRequest] __________

request_type = <class 'google.iam.credentials_v1.types.common.GenerateAccessTokenRequest'>

    @pytest.mark.parametrize("request_type", [
        common.GenerateAccessTokenRequest,
        dict,
    ])
    def test_generate_access_token_rest(request_type):
        client = IAMCredentialsClient(
            credentials=ga_credentials.AnonymousCredentials(),
            transport="rest",
        )
    
        # send a request that will satisfy transcoding
        request_init = {'name': 'projects/sample1/serviceAccounts/sample2'}
        request = request_type(**request_init)
    
        # Mock the http request call within the method and fake a response.
        with mock.patch.object(type(client.transport._session), 'request') as req:
            # Designate an appropriate value for the returned response.
            return_value = common.GenerateAccessTokenResponse(
                  access_token='access_token_value',
            )
    
            # Wrap the value into a proper Response obj
            response_value = Response()
            response_value.status_code = 200
            # Convert return value to protobuf type
            return_value = common.GenerateAccessTokenResponse.pb(return_value)
            json_return_value = json_format.MessageToJson(return_value)
    
            response_value._content = json_return_value.encode('UTF-8')
            req.return_value = response_value
>           response = client.generate_access_token(request)

tests/unit/gapic/credentials_v1/test_iam_credentials.py:1823: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
google/iam/credentials_v1/services/iam_credentials/client.py:724: in generate_access_token
    response = rpc(
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/gapic_v1/method.py:131: in __call__
    return wrapped_func(*args, **kwargs)
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/retry/retry_unary.py:293: in retry_wrapped_func
    return retry_target(
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/retry/retry_unary.py:153: in retry_target
    _retry_error_helper(
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/retry/retry_base.py:212: in _retry_error_helper
    raise final_exc from source_exc
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/retry/retry_unary.py:144: in retry_target
    result = target()
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/timeout.py:120: in func_with_timeout
    return func(*args, **kwargs)
.nox/prerelease_deps/lib/python3.12/site-packages/google/api_core/grpc_helpers.py:76: in error_remapped_callable
    return callable_(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = IAMCredentialsRestTransport._GenerateAccessToken(_session=<google.auth.transport.requests.AuthorizedSession object at ...le.iam.credentials_v1.services.iam_credentials.transports.rest.IAMCredentialsRestInterceptor object at 0x7f0783088f80>)
request = name: "projects/sample1/serviceAccounts/sample2"


    def __call__(self,
            request: common.GenerateAccessTokenRequest, *,
            retry: OptionalRetry=gapic_v1.method.DEFAULT,
            timeout: Optional[float]=None,
            metadata: Sequence[Tuple[str, str]]=(),
            ) -> common.GenerateAccessTokenResponse:
        r"""Call the generate access token method over HTTP.
    
        Args:
            request (~.common.GenerateAccessTokenRequest):
                The request object.
            retry (google.api_core.retry.Retry): Designation of what errors, if any,
                should be retried.
            timeout (float): The timeout for this request.
            metadata (Sequence[Tuple[str, str]]): Strings which should be
                sent along with the request as metadata.
    
        Returns:
            ~.common.GenerateAccessTokenResponse:
    
        """
    
        http_options: List[Dict[str, str]] = [{
            'method': 'post',
            'uri': '/v1/{name=projects/*/serviceAccounts/*}:generateAccessToken',
            'body': '*',
        },
        ]
        request, metadata = self._interceptor.pre_generate_access_token(request, metadata)
        pb_request = common.GenerateAccessTokenRequest.pb(request)
        transcoded_request = path_template.transcode(http_options, pb_request)
    
        # Jsonify the request body
    
>       body = json_format.MessageToJson(
            transcoded_request['body'],
            including_default_value_fields=False,
            use_integers_for_enums=False
        )
E       TypeError: MessageToJson() got an unexpected keyword argument 'including_default_value_fields'

google/iam/credentials_v1/services/iam_credentials/transports/rest.py:325: TypeError
```

--- a/gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2
+++ b/gapic/ads-templates/%namespace/%name/%version/%sub/services/%service/transports/rest.py.j2
@@ -335,7 +335,6 @@ class {{service.name}}RestTransport({{service.name}}Transport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums={{ opts.rest_numeric_enums }}
             )
             {%- endif %}
@@ -346,7 +345,6 @@ class {{service.name}}RestTransport({{service.name}}Transport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums={{ opts.rest_numeric_enums }},
             ))
             {% if method.input.required_fields %}

--- a/gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2
+++ b/gapic/ads-templates/tests/unit/gapic/%name_%version/%sub/test_%service.py.j2
@@ -1201,7 +1201,6 @@ def test_{{ method_name }}_rest_required_fields(request_type={{ method.input.ide
     {% endif %}
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 

--- a/gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2
+++ b/gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/rest.py.j2
@@ -377,7 +377,6 @@ class {{service.name}}RestTransport({{service.name}}Transport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums={{ opts.rest_numeric_enums }}
             )
             {%- endif %}
@@ -388,7 +387,6 @@ class {{service.name}}RestTransport({{service.name}}Transport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums={{ opts.rest_numeric_enums }},
             ))
             {% if method.input.required_fields %}

--- a/gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2
+++ b/gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2
@@ -1090,7 +1090,6 @@ def test_{{ method_name }}_rest_required_fields(request_type={{ method.input.ide
     {% endif %}
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 

--- a/tests/integration/goldens/asset/google/cloud/asset_v1/services/asset_service/transports/rest.py
+++ b/tests/integration/goldens/asset/google/cloud/asset_v1/services/asset_service/transports/rest.py
@@ -807,7 +807,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -887,7 +886,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -896,7 +894,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -974,7 +971,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1053,7 +1049,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1133,7 +1128,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1213,7 +1207,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1289,7 +1282,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1369,7 +1361,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1452,7 +1443,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1461,7 +1451,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1539,7 +1528,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1548,7 +1536,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1621,7 +1608,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1685,7 +1671,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1755,7 +1740,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1764,7 +1748,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1847,7 +1830,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1925,7 +1907,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2001,7 +1982,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2077,7 +2057,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2153,7 +2132,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2228,7 +2206,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -2237,7 +2214,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2314,7 +2290,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2390,7 +2365,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2473,7 +2447,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -2482,7 +2455,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2560,7 +2532,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -2569,7 +2540,6 @@ class AssetServiceRestTransport(AssetServiceTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))

--- a/tests/integration/goldens/asset/tests/unit/gapic/asset_v1/test_asset_service.py
+++ b/tests/integration/goldens/asset/tests/unit/gapic/asset_v1/test_asset_service.py
@@ -7088,7 +7088,6 @@ def test_export_assets_rest_required_fields(request_type=asset_service.ExportAss
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7269,7 +7268,6 @@ def test_list_assets_rest_required_fields(request_type=asset_service.ListAssetsR
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7559,7 +7557,6 @@ def test_batch_get_assets_history_rest_required_fields(request_type=asset_servic
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7752,7 +7749,6 @@ def test_create_feed_rest_required_fields(request_type=asset_service.CreateFeedR
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7999,7 +7995,6 @@ def test_get_feed_rest_required_fields(request_type=asset_service.GetFeedRequest
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8232,7 +8227,6 @@ def test_list_feeds_rest_required_fields(request_type=asset_service.ListFeedsReq
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8474,7 +8468,6 @@ def test_update_feed_rest_required_fields(request_type=asset_service.UpdateFeedR
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8701,7 +8694,6 @@ def test_delete_feed_rest_required_fields(request_type=asset_service.DeleteFeedR
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8926,7 +8918,6 @@ def test_search_all_resources_rest_required_fields(request_type=asset_service.Se
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9222,7 +9213,6 @@ def test_search_all_iam_policies_rest_required_fields(request_type=asset_service
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9515,7 +9505,6 @@ def test_analyze_iam_policy_rest_required_fields(request_type=asset_service.Anal
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9689,7 +9678,6 @@ def test_analyze_iam_policy_longrunning_rest_required_fields(request_type=asset_
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9865,7 +9853,6 @@ def test_analyze_move_rest_required_fields(request_type=asset_service.AnalyzeMov
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10061,7 +10048,6 @@ def test_query_assets_rest_required_fields(request_type=asset_service.QueryAsset
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10315,7 +10301,6 @@ def test_create_saved_query_rest_required_fields(request_type=asset_service.Crea
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10573,7 +10558,6 @@ def test_get_saved_query_rest_required_fields(request_type=asset_service.GetSave
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10808,7 +10792,6 @@ def test_list_saved_queries_rest_required_fields(request_type=asset_service.List
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -11169,7 +11152,6 @@ def test_update_saved_query_rest_required_fields(request_type=asset_service.Upda
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -11400,7 +11382,6 @@ def test_delete_saved_query_rest_required_fields(request_type=asset_service.Dele
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -11624,7 +11605,6 @@ def test_batch_get_effective_iam_policies_rest_required_fields(request_type=asse
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -11819,7 +11799,6 @@ def test_analyze_org_policies_rest_required_fields(request_type=asset_service.An
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -12126,7 +12105,6 @@ def test_analyze_org_policy_governed_containers_rest_required_fields(request_typ
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -12433,7 +12411,6 @@ def test_analyze_org_policy_governed_assets_rest_required_fields(request_type=as
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 

--- a/tests/integration/goldens/credentials/google/iam/credentials_v1/services/iam_credentials/transports/rest.py
+++ b/tests/integration/goldens/credentials/google/iam/credentials_v1/services/iam_credentials/transports/rest.py
@@ -324,7 +324,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -333,7 +332,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -409,7 +407,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -418,7 +415,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -494,7 +490,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -503,7 +498,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -579,7 +573,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -588,7 +581,6 @@ class IAMCredentialsRestTransport(IAMCredentialsTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))

--- a/tests/integration/goldens/credentials/tests/unit/gapic/credentials_v1/test_iam_credentials.py
+++ b/tests/integration/goldens/credentials/tests/unit/gapic/credentials_v1/test_iam_credentials.py
@@ -1837,7 +1837,6 @@ def test_generate_access_token_rest_required_fields(request_type=common.Generate
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -2083,7 +2082,6 @@ def test_generate_id_token_rest_required_fields(request_type=common.GenerateIdTo
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -2331,7 +2329,6 @@ def test_sign_blob_rest_required_fields(request_type=common.SignBlobRequest):
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -2577,7 +2574,6 @@ def test_sign_jwt_rest_required_fields(request_type=common.SignJwtRequest):
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 

--- a/tests/integration/goldens/eventarc/google/cloud/eventarc_v1/services/eventarc/transports/rest.py
+++ b/tests/integration/goldens/eventarc/google/cloud/eventarc_v1/services/eventarc/transports/rest.py
@@ -900,7 +900,6 @@ class EventarcRestTransport(EventarcTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -909,7 +908,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -987,7 +985,6 @@ class EventarcRestTransport(EventarcTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -996,7 +993,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1074,7 +1070,6 @@ class EventarcRestTransport(EventarcTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1083,7 +1078,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1162,7 +1156,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1240,7 +1233,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1318,7 +1310,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1401,7 +1392,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1485,7 +1475,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1570,7 +1559,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1649,7 +1637,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1728,7 +1715,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1807,7 +1793,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1884,7 +1869,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1961,7 +1945,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2038,7 +2021,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2117,7 +2099,6 @@ class EventarcRestTransport(EventarcTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -2126,7 +2107,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2210,7 +2190,6 @@ class EventarcRestTransport(EventarcTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -2219,7 +2198,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -2299,7 +2277,6 @@ class EventarcRestTransport(EventarcTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -2308,7 +2285,6 @@ class EventarcRestTransport(EventarcTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))

--- a/tests/integration/goldens/eventarc/tests/unit/gapic/eventarc_v1/test_eventarc.py
+++ b/tests/integration/goldens/eventarc/tests/unit/gapic/eventarc_v1/test_eventarc.py
@@ -5947,7 +5947,6 @@ def test_get_trigger_rest_required_fields(request_type=eventarc.GetTriggerReques
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -6184,7 +6183,6 @@ def test_list_triggers_rest_required_fields(request_type=eventarc.ListTriggersRe
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -6537,7 +6535,6 @@ def test_create_trigger_rest_required_fields(request_type=eventarc.CreateTrigger
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -6854,7 +6851,6 @@ def test_update_trigger_rest_required_fields(request_type=eventarc.UpdateTrigger
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7095,7 +7091,6 @@ def test_delete_trigger_rest_required_fields(request_type=eventarc.DeleteTrigger
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7351,7 +7346,6 @@ def test_get_channel_rest_required_fields(request_type=eventarc.GetChannelReques
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7588,7 +7582,6 @@ def test_list_channels_rest_required_fields(request_type=eventarc.ListChannelsRe
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -7941,7 +7934,6 @@ def test_create_channel_rest_required_fields(request_type=eventarc.CreateChannel
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8258,7 +8250,6 @@ def test_update_channel_rest_required_fields(request_type=eventarc.UpdateChannel
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8497,7 +8488,6 @@ def test_delete_channel_rest_required_fields(request_type=eventarc.DeleteChannel
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8742,7 +8732,6 @@ def test_get_provider_rest_required_fields(request_type=eventarc.GetProviderRequ
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -8979,7 +8968,6 @@ def test_list_providers_rest_required_fields(request_type=eventarc.ListProviders
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9277,7 +9265,6 @@ def test_get_channel_connection_rest_required_fields(request_type=eventarc.GetCh
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9514,7 +9501,6 @@ def test_list_channel_connections_rest_required_fields(request_type=eventarc.Lis
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -9866,7 +9852,6 @@ def test_create_channel_connection_rest_required_fields(request_type=eventarc.Cr
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10109,7 +10094,6 @@ def test_delete_channel_connection_rest_required_fields(request_type=eventarc.De
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10342,7 +10326,6 @@ def test_get_google_channel_config_rest_required_fields(request_type=eventarc.Ge
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -10642,7 +10625,6 @@ def test_update_google_channel_config_rest_required_fields(request_type=eventarc
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 

--- a/tests/integration/goldens/redis/google/cloud/redis_v1/services/cloud_redis/transports/rest.py
+++ b/tests/integration/goldens/redis/google/cloud/redis_v1/services/cloud_redis/transports/rest.py
@@ -681,7 +681,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -690,7 +689,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -769,7 +767,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -846,7 +843,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -855,7 +851,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -933,7 +928,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -942,7 +936,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1018,7 +1011,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1095,7 +1087,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1174,7 +1165,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1183,7 +1173,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1261,7 +1250,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1340,7 +1328,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1349,7 +1336,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1427,7 +1413,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1436,7 +1421,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))
@@ -1514,7 +1498,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
 
             body = json_format.MessageToJson(
                 transcoded_request['body'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False
             )
             uri = transcoded_request['uri']
@@ -1523,7 +1506,6 @@ class CloudRedisRestTransport(CloudRedisTransport):
             # Jsonify the query params
             query_params = json.loads(json_format.MessageToJson(
                 transcoded_request['query_params'],
-                including_default_value_fields=False,
                 use_integers_for_enums=False,
             ))
             query_params.update(self._get_unset_required_fields(query_params))

--- a/tests/integration/goldens/redis/tests/unit/gapic/redis_v1/test_cloud_redis.py
+++ b/tests/integration/goldens/redis/tests/unit/gapic/redis_v1/test_cloud_redis.py
@@ -3739,7 +3739,6 @@ def test_list_instances_rest_required_fields(request_type=cloud_redis.ListInstan
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -4083,7 +4082,6 @@ def test_get_instance_rest_required_fields(request_type=cloud_redis.GetInstanceR
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -4318,7 +4316,6 @@ def test_get_instance_auth_string_rest_required_fields(request_type=cloud_redis.
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -4613,7 +4610,6 @@ def test_create_instance_rest_required_fields(request_type=cloud_redis.CreateIns
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -4919,7 +4915,6 @@ def test_update_instance_rest_required_fields(request_type=cloud_redis.UpdateIns
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -5147,7 +5142,6 @@ def test_upgrade_instance_rest_required_fields(request_type=cloud_redis.UpgradeI
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -5379,7 +5373,6 @@ def test_import_instance_rest_required_fields(request_type=cloud_redis.ImportIns
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -5608,7 +5601,6 @@ def test_export_instance_rest_required_fields(request_type=cloud_redis.ExportIns
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -5837,7 +5829,6 @@ def test_failover_instance_rest_required_fields(request_type=cloud_redis.Failove
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -6066,7 +6057,6 @@ def test_delete_instance_rest_required_fields(request_type=cloud_redis.DeleteIns
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 
@@ -6292,7 +6282,6 @@ def test_reschedule_maintenance_rest_required_fields(request_type=cloud_redis.Re
     pb_request = request_type.pb(request)
     jsonified_request = json.loads(json_format.MessageToJson(
         pb_request,
-        including_default_value_fields=False,
         use_integers_for_enums=False
     ))
 

